src/pythonic/index.md

```diff
@@ -116,6 +116,23 @@ for conveying what you're doing.
 > Also, make sure you document your exported functions. From experience, working with 
 > libraries with non-existent documentation is not fun. Please don't add to the problem!
 
+The contents of [Cargo.toml](./pythonic/primes/Cargo.toml) are below:
+```
+[package]
+name = "primes"
+version = "0.1.0"
+authors = ["Michael Bryan <michaelfbryan@gmail.com>"]
+
+[dependencies]
+primal = "*"
+libc = "*"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+```
+
+Adding `cdylib` to the `lib` section compiles the crate as a system library
+usable by other languages.
```

Comment:
Well a `cdylib` is a dynamic library, whereas a `rlib` is something kinda like a static library (I think). So the aim is to make a dynamic library here, they're typically a lot easier to use because you don't have to manually link in all the dependencies. Instead the loader will do it when an executable is initially loaded into memory before being executed, and if there are any "linker" errors the thing just won't run.

We may as well remove the `rlib` entry because it'll just add extra confusion.